package-lock.json

```diff
@@ -0,0 +1,4116 @@
+{
```

Comment:
Please remove this file and add it to `.gitignore`. The library was developed prior to NPM 5's wide-spread use and wasn't prepared to block it already.